examples/evaluator/README.md

```diff
@@ -22,11 +22,30 @@ It consists of a [Python
 program](https://github.com/qir-alliance/pyqir/tree/main/examples/evaluator/bernstein_vazirani.py)
   that loads the [compiled
   bitcode](https://github.com/qir-alliance/pyqir/tree/main/examples/evaluator/bernstein_vazirani.bc)
-  and then uses the `NonadaptiveEvaluator`, and a custom `GateLogger` to print out a
+  and then uses the `NonadaptiveEvaluator`, and a `GateLogger` to print out a
   simple log of the quantum gates applied during execution.
 
   The example can be run using python:
 
   ```bash
   python bernstein_vazirani.py
   ```
+
+- **Teleport example**: <br/>
+This example shows how to log the executed gate sequence leveraging
+a supplied result stream. The `NonadaptiveEvaluator`'s `eval` call accepts
```

Comment:
It may be worth calling out that this functionality is intended for testing purposes. It may otherwise confuse some people that are trying to use this for simulation or connecting to hardware.